mautic/mautic
# The API defines Contacts not Contact causing the API to not receive the correct mapping

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ x]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

Rebases PR https://github.com/mautic/mautic/pull/9559 to 5.x branch. 



#### Steps to test this PR:

How to test:

> 1. Set up mautic
> 2. Set up a trial Dynamics 365 account
> 3. Set up the Dynamics CRM Plugin and select contacts to sync
> 
> Without the patch, no fields from the contacts will show up, no contacts can be synced
> With the patch, contacts fields will show and contacts will sync :)
> It's kind of logical because all call towards this function use the plural. I'm not sure where the singular "Contact" would come from but in testing I found that it becomes "Contacts" instead of "Contact", hence this patch.


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.
